src/robot/conf/languages.py

```diff
@@ -110,7 +110,7 @@ def settings(self):
         return settings
 
 
-class En(Language):
+class EN(Language):
```

Comment:
Our class naming convention has followed `TitleCase` format also with abbreviations like `HtmlToXml`. I guess using `EN` instead of `En` would be fine in this context, but it causes problems with language variants like `PT-BR`. I'd prefer them to be in format like `PtBr` and not `PTBR`.